### DISCUSSION TITLE
Mortal wound idea thread

### DISCUSSION THREAD

**Discussion by Lambosaurus** [Lambosaurus/hives-and-torches#58] [Mechanics]
A thread for listing some ideas for mortal wounds

**Comment by Lambosaurus** [comment]
Fracture:
 - Permenantly crippled (half movement, cant maneuver)
 
 This might be too strong

**Reply by Lambosaurus** [reply]
This is **not** too strong. It is inline with some of the other stronger mortal wounds.

**Comment by Lambosaurus** [comment]
Broken hand:
 - Loose use of one hand

**Comment by Lambosaurus** [comment]
Disfigured:
An unsettling wound marks your face
* Disadvantage on diplomacy tests (May be ignored by DM when appropriate)

**Comment by Lambosaurus** [comment]
Addled:
* Disadvantage when making any reactions

**Reply by Lambosaurus** [reply]
I think this is too weak. Need to make it a bit worse.

**Reply by Lambosaurus** [reply]
I think this is obsoleted by concussed.

**Reply by LouderthanThunder** [reply]
Delete this

**Comment by Lambosaurus** [comment]
Weeping wounds:
* When taking a rest, make a medicine test. On a failure you may not restore the normal fatigue.

**Comment by Lambosaurus** [comment]
Missing finger:
* `-2` on attack rolls

**Comment by Lambosaurus** [comment]
Vacant stare:
- On initiative make a will roll. On a fail, your make no action on your first round. (You may hold 1 reaction)

**Comment by Lambosaurus** [comment]
Bleeding eye:
 - Disadvantage on attacks on targets further than 5 feet away

**Comment by Lambosaurus** [comment]
Frostbite:
 - Movement speed reduced by 10 feet.
 - `-1` on attack rolls

**Comment by Lambosaurus** [comment]
Broken rib:
 - Max fatigue reduced by `1d4` + level

**Comment by Lambosaurus** [comment]
Large burns:
 - Fatigue restoration halved (round up)

**Reply by Lambosaurus** [reply]
This is now blanketly weaker than anything that inflicts blighted.

**Comment by Lambosaurus** [comment]
Coughing blood:
 - Disadvantage on stealth
 - Disadvantage on any strength tests

**Comment by Lambosaurus** [comment]
Deep bruises
- `-2` defence

**Comment by LouderthanThunder** [comment]
Dungeon rot 

Requires a series of medical treatments to resolve, otherwise the affected limb can be severed. If the limb is not severed the player risks dying from infection

**Reply by Lambosaurus** [reply]
How about I make a new status effect:
Blighted:
 - Disadvantage on all strength and finesse tests, including attacks.
 - Fatigue healing is halved
 
 Dungeon rot and gut wound can inflict blighted.
 I'm not really sure how to put rules around getting the limb severed.
 Maybe after `1d4 + 1` days of the wound starting, the infection can become so bad that a limb must be amputated.
 This can be delayed by a `1` day with a successful medicine test during a long rest.

**Reply by LouderthanThunder** [reply]
Yup that sounds good. Dungeon Rot is just any infection. Blighted also works

**Comment by LouderthanThunder** [comment]
Concussion. 

Player has reduced fatigue and will for 2-3 days. This mortal wound only kicks in after day one. If the player pushes it and doesn't rest up they lose one int for the rest of the campaign

**Reply by Lambosaurus** [reply]
How about a new status dazed: ( a counterpart to blighted )
 - Disadvantage on intellect and will tests, including spells
 - Unable to take reactions

Concussion inflicts:
 - Dazed
 - If the player misses a long rest, they may loose 1 intellect permanently. (not sure about this)

**Reply by LouderthanThunder** [reply]
From my experience with concussion they usually kick in after the first day. So if becomes a medical check day 0 and if it's made and passed the healer can pick up on the concussion then and there. Day one is when rhe headache and misery starts. I like the new statuses

**Comment by LouderthanThunder** [comment]
The voices 

Caused by suffering a mortal wound from a dark miracle/cursed item or other 'evil' source. 

Player loses one will. In addition to this the player is now hostile towards neutral npcs and loses their bonus to social rolls 

The dm can also use this condition as a plot device to cause the player to do something horrid

**Reply by Lambosaurus** [reply]
I'm pretty opposed to anything that forces a PC's actions

**Reply by LouderthanThunder** [reply]
That's possibly due to the people that dm for you. But fair enough

**Reply by Lambosaurus** [reply]
I think it needs to be an agreement that happens between the player and the DM, and shouldn't be in the stock table.
I feel pretty strong about this one.

**Reply by LouderthanThunder** [reply]
Weak. Anyway I like the idea of spiritual mortal wounds. Or arcane 

To bring in cleaning rituals to sanctify or things like that. Ties in with our miracle ideas and feel 

Possibly arcane magic could leave victims frazzled from shock or something

**Comment by LouderthanThunder** [comment]
Broken

The player will have a long lasting fear of the npc type that caused the wound. In his next fight featuring that race they will be at disadvantage against said race. If they win the combat this condition may be revoked at the dms discretion

**Reply by Lambosaurus** [reply]
This seems balanced enough to me. Just needs a reword and its golden.

**Comment by LouderthanThunder** [comment]
Gutted 

The player has suffered a gut shot. They struggle to eat and digest food and are in constant pain. 

Half healing from rest involving eating 

The player cannot run 

The wound is a bitch to heal without magic and will probably lead to death if not treated

**Reply by Lambosaurus** [reply]
How about:
 - The character is blighted.
 - Movement speed reduced by `10 feet`

**Reply by LouderthanThunder** [reply]
Yup blighted will standardise a few things
